### DISCUSSION TITLE
fail to download mkcert

### DISCUSSION THREAD

**Discussion by HomyeeKing** [FiloSottile/mkcert#442] [Q&A] [answered]
this file 
https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:053f02796ab0165faaabc470cc161559d3ba5062b5e56f6df1bbd46a828f4991?se=2021-08-27T04:55:00Z&

has no public access not permitted
![image](https://user-images.githubusercontent.com/49113249/131074758-83199ceb-dadd-4122-8c9d-9a0a164270df.png)

**Comment by rfay** [comment]
Shouldn't you be downloading a release from https://github.com/FiloSottile/mkcert/releases ?

Not sure what that link may e pointing to.

**Comment by HomyeeKing** [comment]
I'm using homebrew

**Comment by rfay** [comment, marked as answer]
You probably need to upgrade homebrew. It's changed a lot. `brew update`
